app/views/layouts/kss/application.html.erb

```diff
@@ -14,6 +14,7 @@
         <ul>
           <li><%= link_to 'Home', kss.root_path %></li>
           <li><%= link_to 'Styleguide', kss.styleguide_path %></li>
+          <li><%= link_to 'Styleguide New', '/kss/styleguide2' %></li>
```

Comment:
We should sort out this situation.

The three files we will serve include
- the old KSS-only styleguide? Still useful? get rid of it?
- the template of the new styleguide, before it has app HTML in it
- the new live styleguide

We'll need to choose good URLs for these and link them up in the right spots (e.g. readme, etc)
— @DTrejo